### ISSUE TITLE
07_beancount_options_reference.html is hard to read

### ISSUE THREAD

**Issue by berkes** [xuhcc/beancount-docs#1]
The formatting is off. It appears to be a combination of the indentation being lost and the paragraphs not being formatted like the source.

https://docs.google.com/document/d/1_-T_BvDtUjj9M7liZMNSkrL8pgC60TGMBlYCiV1e4ZM/edit

![afbeelding](https://user-images.githubusercontent.com/77059/62490714-4c20b700-b7ca-11e9-940f-6b4748a0e599.png)

And the output: 

![afbeelding](https://user-images.githubusercontent.com/77059/62490778-71152a00-b7ca-11e9-8553-1ccbfa608bb0.png)

If configurable in pandoc, an option would be to use [a definition-list, `DL`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl). 
In the current theme, the following HTML:
```
<dl>
  <dt>option “account_previous_balances” “Opening-Balances"</dt>
  <dd>Leaf name of the equity account used for summarizing previous 
transactions into opening balances.</dd>
</dl>
```

Shows up as:

![afbeelding](https://user-images.githubusercontent.com/77059/62491101-55f6ea00-b7cb-11e9-97e9-a75a331bd276.png)

**Comment by xuhcc** [comment]
Unfortunately this google document has very little formatting and it is hard to parse it correctly. For now I just wrapped all paragraphs in code blocks:

c67350fd24532bffd87978d27804db3f896889f3

**Comment by berkes** [comment]
For now, I'd say this works fine.

Maybe we can get the upstream google document to change it's markup a little, if we encounter more such issues?
I guess. using a header or even bold lines in the google doc would help pandoc distinguish the outcome.

**Comment by xuhcc** [comment]
Yes, in general I would prefer to amend the original documents instead of comlicating the converter. 

In this specific document pandoc sees every single line as a separate paragraph. I think it could be solved by removing unnecessary line breaks in the option descriptions.
